pyproject.toml

```diff
@@ -193,24 +193,27 @@ select = [
     # would exclude:
     # "N806", # probably not worth it
     # "N812" # disallows: import torch.nn.functional as F which is standard
-
+    "RUF",
```

Comment:
isn't this a somewhat random collection of rules - I would rather select the specific ones that we want